src/ApplePieRecipe.java

```diff
@@ -0,0 +1,82 @@
+public class ApplePieRecipe {
+
+    Ingredient butter = new Ingredient(200, "gram", "ongezouten roomboter");
+    Ingredient sugar = new Ingredient(200, "gram", "witte basterdsuiker");
```

Comment:
Goed dat je Engelse namen hebt gebruikt! 

Heel klein dingetje: de witte basterdsuiker heet nu "sugar" en de kristal suiker is "crystalSugar". Er zijn natuurlijk nog veel meer soorten suiker, dus misschien had je meteen de Engelse naam voor witte basterdsuiker kunnen kiezen om zo specifiek mogelijk te zijn.